src/index.html

```diff
@@ -28,8 +28,7 @@
     </script>
     <script src="js/page.js" defer></script>
     <link rel="preload" as="style" href="all.css" onload="rel='stylesheet'">
-    <link rel="preload" crossorigin as="style" href="https://fonts.googleapis.com/css?family=Inconsolata" onload="rel='stylesheet'">
-    <link rel="preconnect" crossorigin href="https://fonts.gstatic.com">
+    <link rel="preload" crossorigin as="font" href="fonts/code-latin.woff2">
```

Comment:
Could we maybe not preload the code font at all? I mean, the initial page doesn't need it.

Just wondering, again, I swear it is my last comment on this PR :)

Reply:
It's good to preload it before it's needed, to reduce 'block' delay. I guess it could be preloaded once an SVG is loaded, but it feels like a bit of micro-optimisation. It's only 11.5kB, it isn't render blocking, and it isn't getting in the way of more important resources.